Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Environment.hs

```diff
@@ -96,6 +96,9 @@ data AppCfg = AppCfg
     snapToRoadSnippetThreshold :: HighPrecMeters,
     minTripDistanceForReferralCfg :: Maybe HighPrecMeters,
     maxShards :: Int,
+    driverPaymentCycleBuffer :: NominalDiffTime,
+    driverPaymentCycleDuration :: NominalDiffTime,
+    driverPaymentCycleStartTime :: NominalDiffTime,
```

Comment:
these three fields can be used in single object, like:
```
DriverPaymentCycleCfg = DriverPaymentCycleCfg
{   buffer :: NominalDiffTime,
    duration :: NominalDiffTime,
    startTime :: NominalDiffTime
}
```

Reply:
By the way shouldn't be these fields in transporter_config table? Because in task description said:

> If there is no entry for DriverPlatfromFee for that day of a driver, create one with ONGOING status where startTime is equal to the paymentDayStartTime config defined inside transporter config,